### PR TITLE
ZKUI-58 Enabling Async Notification should enable versioning

### PR DESCRIPTION
**Pull request by nicolas2bert**
Screen-Recording-2022-01-05-at-4 00 47-PM

### CODE REVIEW

**Comment by naren-rajendran** [comment]
@nicolas2bert I am not sure if it is related..but why is `versioning` either `active` or `inactive` and `async notifications` has `enabled` or `disabled` shouldn't they be the same? is there a difference or reasoning behind this?

**Comment by nicolas2bert** [comment]
Thanks! It is not directly related to this PR but good catch. I created a ticket: https://scality.atlassian.net/browse/ZKUI-60 and will double check with @Cuervino to make sure it is the expected wording.

File: src/react/databrowser/buckets/BucketCreate.jsx [hunk 1]
```diff
@@ -153,6 +154,12 @@ function BucketCreate() {
     return `${option.name} (${locationTypeName})`;
   };
 
+  const matchVersionning = (checked: boolean) => {
```
Comment (miniscruff) [inline src/react/databrowser/buckets/BucketCreate.jsx]:
```suggestion
  const matchVersioning = (checked: boolean) => {
```
One too many N's

**Review by miniscruff — APPROVED** [review]

**Comment by nicolas2bert** [comment]
@bert-e approve

**Comment by nicolas2bert** [comment]
ping

**Review by naren-rajendran — APPROVED** [review]

**Pull request merged**